### ISSUE TITLE
irc colors on slack.

### ISSUE THREAD

**Issue by vergecurrency** [ekmartin/slack-irc#98]
it would be a great enhancement for the bot to pick up irc colors, instead of printing out the color codes in slack. this is useful because i often push commits to our community's main repo, and a git irc bot relays the commit to irc, and it looks great (it uses color), but when slack users see it, its atrocious, because all the color codes are printed instead of picked up (becase obviously slack and irc color codes are different). 

ps: just set this up for my community to connect our irc and slack, and its great. excellent work!

**Comment by ekmartin** [comment]
Definitely. Could implement something similar to [parseText](https://github.com/ekmartin/slack-irc/blob/master/lib%2Fbot.js#L136-L164), but for IRC -> Slack. Would you want to try sending a pull request?
